DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DiagnosticsExtension.Models.ConnectionStringValidator
+{
+    public enum ConnectionStringType
+    {
+        SqlServer,
+        RedisCache,
+        StorageAccount,
+        Http
```

Comment:
What do you have in mind for Http for connection string?

Reply:
simple http get